components/sideMenu/index.js

```diff
@@ -1,10 +1,12 @@
 var transitionEndEventName = require('../../lib/transition-end-name')();
+var network = require('../../lib/network.js');
 var Hammer = require('hammerjs');
 
 module.exports = {
     id: 'side-menu',
     ready: function () {
         var self = this;
+        self.$data.isOnline = network.isOnline;
```

Comment:
Since this is a persistent module (ready only runs one time) you will probably want to re-run this check on the `openSideMenu` just to see if we're still offline